jina/drivers/querylang/queryset/dunderkey.py

```diff
@@ -118,6 +124,8 @@ def dunder_get(_dict, key):
 
     if isinstance(parts[0], int):
         result = guard_iter(_dict)[parts[0]]
+    elif isinstance(_dict, dict):
+        result = _dict[parts[0]]
     else:
         result = getattr(_dict, parts[0])
```

Comment:
whats wrong with accessing Pb via `getattr`? why converting to `dict` is necessary?

Reply:
aha, okay, this is only for `Stuct`

Reply:
yes Struct has a weird representation:

https://github.com/protocolbuffers/protobuf/pull/6451

It may be still possible (did not manage myself) to access it but with real weird syntax, (like adding __fields ...)